applications/blackhole/src/blackhole_util.erl

```diff
@@ -127,8 +127,9 @@ respond_with_authn_failure(Context) ->
 %%--------------------------------------------------------------------
 -spec get_callback_module(ne_binary()) -> atom().
 get_callback_module(Binding) ->
-    case binary:split(Binding, <<".">>) of
-        [M, _] ->
+    case binary:split(Binding, <<".">>, ['global']) of
+        [_|_]=RoutingKey ->
```

Comment:
Why not match `[M|_] -> Mod = special_bindings(M), try...`

The tail of the list, and the whole list (RoutingKey) is unused, so why bring it along unnecessarily?